SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -446,6 +446,12 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 		MakeUnique<SpatialGDK::ActorSystem>(ActorNonAuthSubview, TombstoneActorSubview, this, &TimerManager, Connection->GetEventTracer());
 	ClientConnectionManager = MakeUnique<SpatialGDK::ClientConnectionManager>(SystemEntitySubview, this);
 
+	const SpatialGDK::FSubView& DebuggerSubView = Connection->GetCoordinator().CreateSubView(
+		IsServer() ? SpatialConstants::ACTOR_AUTH_TAG_COMPONENT_ID : SpatialConstants::SPATIAL_DEBUGGING_COMPONENT_ID,
```

Comment:
DEBUGGING component isn't a tag, any advantage of using it over ACTOR_NON_AUTH_TAG_COMPONENT_ID ?

Reply:
No need for a filter I guess?.. @jacquese has a similar point here: https://github.com/spatialos/UnrealGDK/pull/2869#discussion_r568887462
The first option (specifying `ACTOR_NON_AUTH_TAG_COMPONENT_ID` + a filter on clients) seems better to me, but seems to achieve the very same thing as using `SPATIAL_DEBUGGING_COMPONENT_ID` on clients with more involvement by querying more entities and them filtering them. Idk.

Reply:
there won't be more entities queried necessarily, as the subview doesn't do any querying of its own. It potentially complicates the logic for an entity becoming complete, with the upside that if we want to change this component/add new components to this system we can do so without worrying about affecting this subview's entities, and also have it explicit that this subview cares about actors you are interested in.

Reply:
Done!